### PR TITLE
Add ng2 example

### PR DESCRIPTION
**Pull request by perryjr**
I caught @bryanbraun 's great talk on animation last week and was inspired  to add an example using Angular 2.

### CODE REVIEW

**Comment by bryanbraun** [comment]
@perryjr, thanks for your PR.

I've been thinking about this PR, and these "framework" type of examples (Angular, React, Ember, etc).

They aren't exactly animation techniques themselves, so these examples have a slightly different purpose. Instead of "comparing animation approaches" they teach "how to create animations within a given framework". Still educational, but different.

I think there's a place for having these comparisons but maybe not mixed in with the other purely animation approaches on the site. I'm thinking about updating the UI to have a place for framework specific comparisons. This would help with other problems, like which code do I display in the code-view when the app is made from a collection of files?

In short, until I get the UI updated I'll probably to leave this PR unmerged.

**Comment by bryanbraun** [comment]
I'm going to close this for now. If I get to the point where I'm looking to add more framework-style examples, I'll ping you back on this issue and we can reopen/resubmit this one.
